lib/active_fedora/orders/builder.rb

```diff
@@ -9,6 +9,9 @@ def self.define_readers(mixin, name)
       mixin.redefine_method("#{name.to_s.gsub("_proxies","").pluralize}") do
         association(name).target_reader
       end
+      mixin.redefine_method("#{name.to_s.gsub("_proxies","").pluralize}=") do |nodes|
```

Comment:
Worth putting `name.to_s.gsub('_proxies', '').pluralize` behind a private method so that both the setter and getter here can both use it?